tests/test_notify_slack.py

```diff
@@ -113,6 +131,54 @@ def should_not_notify_slack_when_step_function_running(webhook_client_mock: Magi
         webhook_client_mock.assert_not_called()
 
 
+@patch("backend.notify_status_update.task.WebhookClient.send")
+@patch("backend.notify_status_update.task.get_import_status_given_arn")
+def should_not_notify_slack_when_step_function_failed(
```

Comment:
Is this title accurate? The assertion is that `webhook_client_mock` *was* called. If it is accurate, why would we not want to notify on failure?